src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -400,8 +401,9 @@ protected Future<Map<String, List<Piece>>> storeUpdatedPieceRecords(Map<String,
       .ofValues(piecesGroupedByPoLine)
       .flatMap(List::stream)
       .filter(this::isSuccessfullyProcessedPiece)
-      .map((Piece piece) -> storeUpdatedPieceRecord(piece, requestContext))
-      .collect(Collectors.toList());
+      .map(piece -> piece.withStatusUpdatedDate(new Date()))
```

Comment:
What if status was not updated in this case, for example just some another field in piece updated?

Reply:
checkin and receiving endpoints is used to transite piece to another status